tests/LexerTest.php

```diff
@@ -268,6 +276,15 @@ public function testStringWithEscapedDelimiterProducingDeprecation(string $templ
 
     public static function getStringWithEscapedDelimiterProducingDeprecation()
     {
+        yield [
+            <<<'EOF'
+            {{ 'Ymd\THis' }}
```

Comment:
this case is exactly the same one that the next case, so no need to add it.

Reply:
Removed it.